HPXMLtoOpenStudio/measure.rb

```diff
@@ -1317,6 +1319,8 @@ def self.add_hot_water_and_appliances(runner, model, weather, spaces)
 
     # Add combi-system EMS program with water use equipment information
     Waterheater.apply_combi_system_EMS(model, @hpxml.water_heating_systems, plantloop_map)
+
+    return plantloop_map[sys_id]
```

Comment:
This currently just returns the last plant loop, which isn't what you want when there are multiple water heaters in a dwelling unit.